File: 1426-find-n-unique-integers-sum-up-to-zero/1426-find-n-unique-integers-sum-up-to-zero.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> res;
        short s = 1;
        while(res.size() < n - 1)
        {
            res.push_back(s);
            res.push_back(-s);
            s++;
        }
        if(res.size() < n)
        {
            res.push_back(0);
        }
        return res;
    }
};

File: 1426-find-n-unique-integers-sum-up-to-zero/1426-find-n-unique-integers-sum-up-to-zero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        res = []
        s = 1
        while len(res) < n - 1:
            res.append(s)
            res.append(-s)
            s += 1
        if len(res) < n:
            res.append(0)
        return res